CMakeModules/Shaders.cmake

```diff
@@ -11,11 +11,9 @@
 
 # find_package(bgfx REQUIRED COMPONENTS shaderc)
 
-# get_target_property(BGFX_INCLUDE_PATH bgfx::bgfx INTERFACE_INCLUDE_DIRECTORIES)
-# set(BGFX_SHADER_INCLUDE_PATH ${BGFX_INCLUDE_PATH}/bgfx)
-
-add_executable(bgfx::shaderc ALIAS shaderc)
-set(BGFX_SHADER_INCLUDE_PATH ${bgfx_SOURCE_DIR}/bgfx/src)
+get_target_property(BGFX_INCLUDE_PATH bgfx::bgfx INTERFACE_INCLUDE_DIRECTORIES)
+list (GET BGFX_INCLUDE_PATH 0 BGFX_INCLUDE_PATH_1) # bgfx::bgfx exports include directory twice?
+set(BGFX_SHADER_INCLUDE_PATH ${BGFX_INCLUDE_PATH_1}/bgfx)
```

Comment:
Can you explain what `BGFX_INCLUDE_PATH_1` is? Why the numbered suffix?
Again, the include path is needed if debuggin bgfx source.

Reply:
bgfx.cmake exposes `BGFX_INCLUDE_PATH` as a LIST, so it has two entries, when passed to shaderc it doesn't understand a list and just errors - so this simply gets the first entry

Reply:
Why not export `BGFX_SHADER_INCLUDE_PATH` in bgfx.cmake That's the logical place to have it. openblack's use of the shader include files is not unique to its application. Every user of bgfx would need that include.